server/graphql/v2/object/TransferWise.ts

```diff
@@ -74,7 +74,7 @@ export const TransferWise = new GraphQLObjectType({
       args: {
         ignoreBlockedCurrencies: {
           type: GraphQLBoolean,
-          description: 'Ignores blocklisted currencies, used to generate the bank information form for manual payments',
+          description: 'Ignores blacklisted currencies, used to generate the bank information form for manual payments',
```

Comment:
We're moving out of terms like blacklist/whitelist and master/slave.
```suggestion
          description: 'Ignores blocked currencies, used to generate the bank information form for manual payments',
```

Reply:
Ah, yes thanks for pointing it out. 🐼